### ISSUE TITLE
Error in Loss Function

### ISSUE THREAD

**Issue by Toku11** [JusperLee/Conv-TasNet#29]
This line https://github.com/JusperLee/Conv-TasNet/blob/9eac70d28a5dba61172ad39dd9fb90caa0d1a45f/Conv-TasNet_lightning/Loss.py#L48 
 could have two different outputs given your test:

if we continue with 
```
if __name__ == "__main__":
    ests = torch.randn(4,320)
    egs = torch.randn(4,320)
    loss = Loss()
    print(loss.compute_loss(ests, egs))
```

N will be length of audio 320

however if we have a list of outputs (two speakers)

```

if __name__ == "__main__":
    ests = [torch.randn(4,320),torch.randn(4,320)]
    egs = [torch.randn(4,320),torch.randn(4,320)]
    loss = Loss()
    print(loss.compute_loss(ests, egs))
```

N will be batch size.

Could you tell me please which one is the correct approach?

I guess your test is wrong and it should be 

```
if __name__ == "__main__":
    ests = [torch.randn(4,320)]
    egs = [torch.randn(4,320)]
    loss = Loss()
    print(loss.compute_loss(ests, egs))
```

Could you validate this?
